packages/particle-cli/package.json

```diff
@@ -1,42 +1,23 @@
 {
-  "private": true,
```

Comment:
Will zeroing out this file prevent publishing the `particle-cli`? Is this intended to become a template for a new projects package.json?

Reply:
This was not my intent. We may be migrating particle-cli to generators-particle-cli